consul/check.py

```diff
@@ -224,6 +224,14 @@ def _cull_services_list(self, services, service_whitelist, max_services=MAX_SERV
 
         return services
 
+    def _get_service_tags(self, service, tags):
+        service_tags = []
+
+        for tag in tags:
+            service_tags.append('consul_{0}_service_tag:{1}'.format(service, tag))
+
+        return ['consul_service_id:{0}'.format(service)] + service_tags
```

Comment:
Nit: you could do `service_tags = ['consul_service_id:{0}'.format(service)]` on the first line of the func and just `return service_tags` at the end to avoid the `+` operation.